users/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('profile', views.profile, name='Users-profile'),
+    path('login/', views.login, name='Users-login'),
+    path('signin/', views.signin, name='Users-signin'),
```

Comment:
Sign in and login means the same thing, instead of sign in type sign up or register.